compiler/test/suites/modules.re

```diff
@@ -0,0 +1,116 @@
+open Grain_tests.TestFramework;
+open Grain_tests.Runner;
+
+describe("modules", ({test, testSkip}) => {
+  let test_or_skip =
+    Sys.backend_type == Other("js_of_ocaml") ? testSkip : test;
+
+  let assertSnapshot = makeSnapshotRunner(test);
+  let assertCompileError = makeCompileErrorRunner(test);
+  let assertRun = makeRunner(test_or_skip);
+  let assertFileRun = makeFileRunner(test_or_skip);
+
+  assertSnapshot(
+    "smallest_module",
+    {|
+      module Foo {
+        void
+      }
```

Comment:
This is smallest submodule, yeah?

Reply:
Yup!

Reply:
Okay, suggestion to rename the test above.